addons/scratchr2/scratchr2_comments.css

```diff
@@ -118,7 +118,7 @@
   border-radius: 8px;
 }
 #comments .comment .info .name a {
-  color: var(--darkWww-link, #4d97ff);
+  color: var(--darkWww-link, #855cd6);
 }
 #comments .comment .info .name a:hover {
   color: var(--darkWww-link-hover, #4280d7);
```

Comment:
This and a bunch of other link-hover colors need to be changed to purple. Look for `#4280d7`.

Reply:
The color Scratch will use for hovered links is `#7854c0`.